src/truncated/normal.jl

```diff
@@ -27,56 +27,57 @@ modes(d::Truncated{Normal{T},Continuous}) where {T <: Real} = [mode(d)]
 # do not export. Used in mean
 # computes mean of standard normal distribution truncated to [a, b]
 function _tnmom1(a, b)
+    mid = float(middle(a, b))
     if !(a ≤ b)
-        return oftype(middle(a, b), NaN)
+        return oftype(mid, NaN)
     elseif a == b
-        return middle(a, b)
+        return mid
     elseif abs(a) > abs(b)
         return -_tnmom1(-b, -a)
     elseif isinf(a) && isinf(b)
-        return zero(middle(a, b))
+        return zero(mid)
     end
-    Δ = (b - a) * middle(a, b)
+    Δ = (b - a) * mid
+    a′ = a * invsqrt2
```

Comment:
@devmotion, I'm curious if you have any thoughts on this approach vs. `a / sqrt2`. They seem to be extremely close but not exactly identical (but close enough to not affect the tests). Similar situation for the division of things by `sqrthalfπ` vs. defining a separate irrational for sqrt(2/π) and multiplying.

Reply:
Hm no, to me both approaches seem fine. I guess if an irrational of the inverse exists, multiplication might be faster and therefore preferable.

Reply:
That was my thought as well, just wasn't sure whether there was something I might be missing in terms of potential loss of precision or things like that